pkg/tiller/release_server.go

```diff
@@ -251,6 +251,16 @@ func (s *ReleaseServer) renderResources(ch *chart.Chart, values chartutil.Values
 		return nil, nil, "", fmt.Errorf("Chart incompatible with Tiller %s", sver)
 	}
 
+	cap, ok := values["Capabilities"].(*chartutil.Capabilities)
```

Comment:
Yes, you are right. I think we can judge if there is k8sversion first. I will change it next week. Thank you for your review.